auto_trainer/function.yaml

```diff
@@ -14,10 +14,10 @@ spec:
     commands: []
-    code_origin: https://github.com/mlrun/functions.git#376ff4637a654c1c9035c24df4f2b9d73d34d192:/Users/yonatanshelach/yoni/projects/functions/auto_trainer/auto_trainer.py
-    origin_filename: /Users/yonatanshelach/yoni/projects/functions/auto_trainer/auto_trainer.py
+    code_origin: https://github.com/yonishelach/functions.git#562773f801b5073bff0fa6b58c13c110c02753eb:/User/functions/auto_trainer/auto_trainer.py
```

Comment:
is it possible to change it back to code origin from mlrun/functions instead of yonishelach/functions?

Reply:
not really..